### PR TITLE
8: Resolve npm audit errors

### PR DESCRIPTION
**Pull request by doug-wade**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,555 @@
...
+      "dev": true
+    },
+    "ylru": {
+      "version": "1.2.1",
+      "resolved": "https://registry.npmjs.org/ylru/-/ylru-1.2.1.tgz",
+      "integrity": "sha512-faQrqNMzcPCHGVC2aaOINk13K+aaBDUPjGWl0teOXywElLjyVAB6Oe2jj62jHYtwsU49jXhScYbvPENK+6zAvQ==",
+      "dev": true
+    }
+  }
+}
```
Comment (haoxins) [inline package-lock.json]: pls add a `.npmrc` with `package-lock=false`

File: package.json [hunk 2]
```diff
@@ -11,13 +11,13 @@
   "license": "MIT",
   "repository": "koajs/trace",
   "dependencies": {
-    "debug": "*"
+    "debug": "^4.1.0"
   },
   "devDependencies": {
-    "mz": "1",
+    "bluebird": "2",
```
Comment (haoxins) [inline package.json]: just remove this :)

File: index.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ module.exports = function (app) {
     var Debug = require('debug')
 
     app.instrument(function (context, event, date, args) {
+      debugger;
```
Comment (haoxins) [inline index.js]: ?

**Comment by doug-wade** [comment]
Thanks for the feedback @coderhaoxin!  I've resolved them all (albeit slowly -- my apologies).

**Review by haoxins — APPROVED** [review]

**Comment by haoxins** [comment]
will release a new version to NPM tomorrow :).

**Comment by haoxins** [comment]
`1.0.1`

**Pull request merged**